pallets/bitacross/src/lib.rs

```diff
@@ -152,10 +152,10 @@ pub mod pallet {
 			pub_key: PubKey,
 		) -> DispatchResultWithPostInfo {
 			let tee_account = T::TEECallOrigin::ensure_origin(origin)?;
-			Vault::<T>::try_mutate(tee_account, |v| {
+			Vault::<T>::try_mutate(tee_account.clone(), |v| {
```

Comment:
I think we can use `&tee_account` instead of `tee_account.clone()`

Reply:
We need to clone it now as `tee_account` is moved when used in the closure